lib/modules/newCommentCount.js

```diff
@@ -336,6 +341,9 @@ addModule('newCommentCount', function(module, moduleID) {
 	}
 
 	function handleButton(threadid, action) {
+		if (module.options.showSubscribeButton.value == true){
```

Comment:
Equivalency checks to true aren't necessary, since it can only be `true` or `false`, and false wouldn't trigger it. A simple `if (module.options.showSubscribeButton.value)` would do.